src/ol/webgl.js

```diff
@@ -105,168 +55,12 @@ export const STATIC_DRAW = 0x88E4;
  */
 export const DYNAMIC_DRAW = 0x88E8;
 
-
-/**
- * @const
- * @type {number}
- */
-export const CULL_FACE = 0x0B44;
-
-
-/**
- * @const
- * @type {number}
- */
-export const BLEND = 0x0BE2;
-
-
-/**
- * @const
- * @type {number}
- */
-export const STENCIL_TEST = 0x0B90;
-
-
-/**
- * @const
- * @type {number}
- */
-export const DEPTH_TEST = 0x0B71;
-
-
-/**
- * @const
- * @type {number}
- */
-export const SCISSOR_TEST = 0x0C11;
-
-
-/**
- * @const
- * @type {number}
- */
-export const UNSIGNED_BYTE = 0x1401;
```

Comment:
You should keep `UNSIGNED_INT`, `UNSIGNED_BYTE` and `UNSIGNED_SHORT` too, as they can be used by the helper API here:
https://github.com/openlayers/openlayers/blob/fac16d75c146c8a1ce88bc15c8d445dac9f461c4/src/ol/webgl/Helper.js#L704-L722

Reply:
done